test/helpers.ts

```diff
@@ -13,7 +13,7 @@ export function useEnvironment(fixtureProjectName: string) {
   beforeEach("Loading hardhat environment", function () {
     process.chdir(path.join(__dirname, "fixture-projects", fixtureProjectName));
     process.env.HARDHAT_NETWORK = networkName;
```

Comment:
@fvictorio I don't remember exactly why doing this:

```ts
process.env.HARDHAT_NETWORK = networkName;
```

is required for testing (my tests work also without it). Is this still something required from Hardhat (I saw it in some of your tests as well)...

Reply:
I'm pretty sure it's not necessary. Our own tests use it because we test some things in `localhost`, and this is how we select the network (it's equivalent to `--network`). But if you are always using the in-process Hardhat network, then you can drop this.